www/main/menu.js

```diff
@@ -172,10 +172,9 @@ const hukamnamaButton = h(
     'a.hukamnama-button',
     {
       onclick: () => {
-        getJSON('https://api.banidb.com/hukamnama/today', (error, response) => {
+        getJSON('https://api.banidb.com/v2/hukamnamas/today ', (error, response) => {
```

Comment:
We should only call this if the user has an internet connection. options are to 

- either hide the button if they are offline or <- my preference. 
- show some message that they must be online, or 
- pick a random shabad.

Reply:
I don't prefer hiding because that might result in the user trying to find "where did the hukamnama feature go?" But I will defer that decision to @maneetpaul veerji.

Reply:
another would be to make it "disabled" if user is offline. we'll need to do the same for the notifications bell.